command/operator_api.go

```diff
@@ -0,0 +1,446 @@
+package command
+
+import (
+	"crypto/tls"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-cleanhttp"
+	"github.com/hashicorp/nomad/api"
+	"github.com/posener/complete"
+)
+
+type OperatorAPICommand struct {
+	Meta
+
+	verboseFlag bool
+	method      string
+	body        io.Reader
+}
+
+func (*OperatorAPICommand) Help() string {
+	helpText := `
+Usage: nomad operator api [options] <path>
+
+  api is a utility command for accessing Nomad's HTTP API and is inspired by
+  the popular curl command line program. Nomad's operator api command populates
```

Comment:
```suggestion
  the popular curl command line tool. Nomad's operator api command populates
```

> command line tool and library

how curl refers to itself 

https://curl.se/